### ISSUE TITLE
Message input field: Cannot exit image preview dialog using keyboard only

### ISSUE THREAD

**Issue by kangabell** [danny-avila/LibreChat#5528]
After adding an image to the chat and opening the preview dialog for it, there is no apparent way to exit with a keyboard only. Focus is trapped inside the dialog and the only way to exit is to refresh the browser page and start over.

## Test Environment
http://159.89.83.1/c/new

## WCAG Criteria
2.1.2 No Keyboard Trap - A

## Testing Method
Manual - VO/Safari/MacOS

## Screen Recording
https://share.zight.com/rRudNmQN
In this video, after opening the image preview, I try to get out of it using keyboard only (tab forward, tab backward, hit Enter key, hit Escape key, etc.), but can’t. Trapped!

## Recommended Action
Add a keyboard-accessible close button to this dialog. Additionally, it is recommended that the Escape key can be used to close the dialog as well.

**Comment by kangabell** [comment]
Update: I re-tested this and the Escape key is working now to dismiss the dialog. It is still highly recommended to have a visible keyboard-accessible button to close the modal, but this is no longer a keyboard trap. Closing.
